cli/Makefile

```diff
@@ -1,3 +1,4 @@
+.PHONY: turbo
```

Comment:
We do, Make doesn't always rebuild because the task turbo creates a turbo file, which makes Make think that it doesn't have to run again. This makes it run no matter what.

Reply:
Ah, I think I see the problem, I ran into this as well. The problem is not that the file already exists, it's that the dependency glob on `.turbo` doesn't handle nested packages. I think something like [this](https://github.com/vercel/turborepo/pull/887/files#diff-8ac507b1c662cdf937055798ca13ec20d9c57c4719e5f6394b91fe02bca3fa2eR11) is a more proper solution.

Reply:
It might be a bit of both, I added PHONY and it wasn't giving me anymore problems.

Reply:
I think we want the behavior without `PHONY`. If nothing has changed, we shouldn't rebuild. We *should* base that calculation on the correct set of files. So if that calculation is incorrect, we should fix it.

Reply:
Make doesn't do hashing, it assumes that everything is up to date because a file named turbo exists. We should technically be deleting turbo before each build, PHONY just allows for no caching without deleting before each build.

Reply:
`make` uses file modification time comparisons between dependencies and output files. So as long as we properly enumerate the input files, we should not need `PHONY`, and in fact we will get caching (which we want) out of not having `PHONY`.